npm/iptm/helper.go

```diff
@@ -13,10 +13,10 @@ func getAllChainsAndRules() [][]string {
 		getAzureNPMAcceptChainRules,
 		getAzureNPMIngressChainRules,
 		getAzureNPMIngressPortChainRules,
-		getAzureNPMIngressFromChainRules,
```

Comment:
We still need these chains no?

Reply:
yeah we will need the chains, this is just removing the default RETURN rules added in these chains

Reply:
Shall we change the function name then? GetAllChainsAndRules is misleading